engine/tester/config_parse.c

```diff
@@ -2110,18 +2280,28 @@ alloc_and_get_run_item(xmlNodePtr node, tester_cfg *cfg, unsigned int opts,
     else if ((~opts & TESTER_RUN_ITEM_SERVICE) &&
              xmlStrcmp(node->name, CONST_CHAR2XML("package")) == 0)
     {
+        if (opts & TESTER_RUN_ITEM_TEMPLATE)
+        {
+            ERROR("Package templates are not supported");
+            return TE_RC(TE_TESTER, TE_EINVAL);
+        }
         p->type = RUN_ITEM_PACKAGE;
         rc = get_package(node, cfg, session, &p->u.package, p);
         if (rc != 0)
             return rc;
     }
-    else
+    else if ((~opts & TESTER_RUN_ITEM_TEMPLATE) ||
+             (opts & TESTER_RUN_ITEM_TEMPLATE &&
+              xmlStrcmp(node->name, CONST_CHAR2XML("arg")) != 0))
     {
         ERROR("The first element '%s' in run item is incorrect",
               XML2CHAR(node->name));
         return TE_RC(TE_TESTER, TE_EINVAL);
     }
-    node = xmlNodeNext(node);
+
+    if ((~opts & TESTER_RUN_ITEM_TEMPLATE) ||
```

Comment:
It took too much time for me to understand what's going on here. IMHO it would be better to move
    node = xmlNodeNext(node);
in script/session/package branches above.